### ISSUE TITLE
[Retail Bug Report] <Keybinds not working after using a portal or entering an instance>

### ISSUE THREAD

**Issue by ajamcek** [tukui-org/ElvUI#489]
### What is the issue you are having?

After the latest patch, almost anytime (maybe every other time) I use a portal or enter an instance most of my keybinds stop working. I can still use my movement keyes (WSAD and space), keybind "1" and "shift-middle mouse buttton", but all the rest are not working. I can click them on my action bars and it works, just the keybinds don't work - the icon gets highlighted, but the spell doesn't work and I can hear the sound of the unsuccessful key clicking. The only way to fix this is by reloading the game. I only get this issue while having Elvui enabled, not with the default UI. I have disabled all my other addons and am still experiencing this issue, so I'm certain it's Elvui's problem. I'm using the version 12.98.

### What is the expected behavior?

Pressing designated keybinds should use the spells associated with them even after using portals etc.

### What actually happened?

The spell icon hets highlighted, but the spell doesn't get used. Instead I can only hear the unsuccessful click sound.

### Suggested Solution/Workaround

The only workaround I've found is to reload the game, however the issue will repeat itself the next time I'll use a portal or enter into a dungeon.

### Errors

```shell
No errors get printed, I've tried using the command /luaerror on but it does nothing.
```

### ElvUI Changelog & Troubleshooting

- [X] I have confirmed that I have read the [Changelog](https://www.tukui.org/download.php?ui=elvui&changelog).
- [X] I have verfied ElvUI is up to date by checking /estatus.
- [X] I have verfied my issue doesn't exist in the ElvUI Issue Tracker.
- [ ] I have not read step six and I am just clicking all boxes.
- [X] I have verfied my issue doesn't exist in the default UI.
- [X] I have disabled all addons except the two called 'ElvUI' and 'ElvUI OptionsUI' by using `/edebug on`? This is important in order to rule out that any of your other addons are causing the issue. If you manage to pinpoint a conflict between ElvUI and another addon, make sure you include that information.

**Comment by Zeron85** [comment]
Getting this also. i'm using the new blizzard bars. 

The sound is the same sound that the game menu makes when appearing

**Comment by Phko92** [comment]
Can confirm that I am also getting this bug.
I am also using default blizz action bars, action bars for elvui are disabled.

Very annoying if you get ported for a raid fight and can't press any buttons.

14x [ADDON_ACTION_FORBIDDEN] AddOn 'ElvUI' tried to call the protected function 'UseAction()'.
[string "@Interface/AddOns/!BugGrabber/BugGrabber.lua"]:480: in function <Interface/AddOns/!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function UseAction'
[string "@Interface/FrameXML/SecureTemplates.lua"]:364: in function handler'
[string "@Interface/FrameXML/SecureTemplates.lua"]:690: in function <Interface/FrameXML/SecureTemplates.luwwwwwa:672>
[string "@Interface/FrameXML/SecureTemplates.lua"]:704: in function <Interface/FrameXML/SecureTemplates.lua:697>
[string "@Interface/FrameXML/SecureTemplates.lua"]:746: in function SecureActionButton_OnClick'
[string "@Interface/FrameXML/ActionButton.lua"]:109: in function TryUseActionButton'
[string "@Interface/FrameXML/MultiActionBars.lua"]:22: in function `MultiActionButtonUp'
[string "MULTIACTIONBAR6BUTTON10"]:4: in function <[string "MULTIACTIONBAR6BUTTON10"]:1>

**Comment by Zeron85** [comment]
i've found using the ElvUI bars gets round the problem, tho it does introduce a new one, its not limiting gameplay

**Comment by InterwovenWasTaken** [comment]
I can confirm the same bug.
1) It occurs always when I load a mythic dungeon and start a key, always when I load a dungeon through dungeon finder, and sometimes when I load a new area or change talents/specialization.
2) It occurs with 'ElvUI' and 'ElvUI OptionsUI' active and no other addons.
3) It occurs when I'm using the Blizzard action bars with the ElvUI action bars disabled.
4) It occurs with ElvUI version 12.99

Using /reload clears the bug, but only until part 1 happens again

**Comment by gz2k2** [comment]
same problem here, with default blizzard action bars and dominos. elvui action bars are disabled

**Comment by LivM00re** [comment]
still present with version 13.07

**Comment by kodewdle** [comment]
should be fixed in 13.08 lmk (on discord, not here) 🌼
